base/libgit2/blob.jl

```diff
@@ -9,6 +9,12 @@ function rawcontent(blob::GitBlob)
     copy(unsafe_wrap(Array, ptr, (length(blob),), false))
 end
 
+"""
+Fetches the contents of the `GitBlob` `blob`. If the `blob` contains
+binary data (which can be determined using [`isbinary`](@ref)) this
+function will throw an error. Otherwise, returns a `String`
```

Comment:
Perhaps the first and third sentences could be fused / simplified, particularly with benefit of a signature as `ararslan` suggests? E.g.

```julia
   content(blob::GitBlob) -> String

Fetch the contents of `blob`. If `blob` contains binary data (which can be determined using [`isbinary`](@ref)), this function will throw an error.
```

Could further simplify `, this function will throw an error` to `, throw an error` or even `, error.`?